msteams-platform/graph-api/meeting-transcripts/overview-transcripts.md

```diff
@@ -1,54 +1,51 @@
 ---
```

Comment:
nit: rename this file to overview-transcripts-recordings?

Reply:
not necessary to change and if we update the link, it will leads to links broken.